### ISSUE TITLE
Captured video is very dark

### ISSUE THREAD

**Issue by blackears** [freemocap/freemocap#564]
I'm trying to use the built in camera capture feature on my new Lenovo Laptop running Windows 11.  The picture it is showing in the Cameras tab is displaying a very dark version of what the camera is capturing.  I tried changing the Exposure setting, but it did not affect the image being displayed in the window.

**Comment by philipqueen** [comment]
We have the defaults set to very dark because it often leads to much better tracking from our pose estimation that using videos that look good for humans. However, you should be able to change this by changing the exposure setting and then clicking the "Apply settings to cameras" button. Is this what you tried and wouldn't work?

**Comment by blackears** [comment]
Yes, I changed the exposure and clicked that button and the display in the window stayed dark.  It started with an exposure of -7. I tried both 0 and -20, but they both seemed equally dark.

**Comment by philipqueen** [comment]
Great thank you for clarifying. We've had this issue with mac webcams before, but it appears it may be cross platform with integrated laptop webcams. I'll look into a fix for this
